frontend/src/common/styles.css

```diff
@@ -0,0 +1,12 @@
+.button {
```

Comment:
is this style for a general use or is it specific for a success or cancel button?

Reply:
general use, so i move it to a common